src/main/java/executor/service/utils/StepAction.java

```diff
@@ -1,7 +1,7 @@
 package executor.service.utils;
 
 public enum StepAction {
-    CLICK_CSS("clickCss");
+    CLICK_CSS("clickCss"),SLEEP("sleep");;
```

Comment:
`CLICK_CSS("clickCss"),SLEEP("sleep");;` - this line of code has two semicolons, not really critical tho